test/controllers/recirculating_infusion_mash_systems_controller_test.rb

```diff
@@ -1,7 +1,47 @@
 require 'test_helper'
 
 class RecirculatingInfusionMashSystemsControllerTest < ActionController::TestCase
-  # test "the truth" do
-  #   assert true
+  setup do
+    @user = users(:ohm)
+    log_in_as(@user)
+    @rhizome = rhizomes(:zeus)
+    @rims = recirculating_infusion_mash_systems(:rims_two)
+    @rims.set_rims
+    @rims.set_rhizome
```

Comment:
Similarly, this looks like the controller method. If you're trying to set ```@rims``` 's Rhizome to one of the ones defined in your fixtures, you can do ```@rims.rhizome = @rhizome``` and save it, or I think you can add the references into the YAML files directly (you're doing that with Thermostats, right?).